modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/DefaultCodegen.java

```diff
@@ -904,6 +904,10 @@ public CodegenParameter fromParameter(Parameter param, Set<String> imports) {
       p.required = param.getRequired();
     p.jsonSchema = Json.pretty(param);
 
+    if(param instanceof QueryParameter) {
+        p.defaultValue = ((QueryParameter)param).getDefaultValue();
+    }
+
```

Comment:
@wing328 Yes you are correct, this PR only currently covers query params (not form or header).